.github/workflows/security.yaml

```diff
@@ -27,3 +27,13 @@ jobs:
         uses: edplato/trufflehog-actions-scan@0af17d9dd1410283f740eb76b0b8f6b696cadefc  # v0.9
         with:
           scanArguments: "--regex --entropy=False --exclude_paths .github/exclude-patterns.txt --max_depth=1"
+  checkov-secrets:
```

Comment:
can it replace detect-secrets run ? 
in checkov we're using it..

Reply:
yeah, that's the goal, but after I added all the skip comments 😄